src/js/timeline.js

```diff
@@ -84,7 +53,7 @@ function getTL(){
 
 document.addEventListener("DOMContentLoaded", () => {
   getTL();
-  document.getElementById("refreshTL").addEventListener("click", () => {
+  document.getElementById('refreshTL').addEventListener("click", () => {
```

Comment:
can you convert this entire piece to react as well? You may need to change your implementation to create a proper component